dapr_agents/workflow/utils/grpc.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+import logging
+from typing import Optional, Sequence
+
+from dapr_agents.agents.configs import WorkflowGrpcOptions
+
+logger = logging.getLogger(__name__)
+
+
+def apply_grpc_options(options: Optional[WorkflowGrpcOptions]) -> None:
+    """
+    Patch Durable Task's gRPC channel factory with custom message size limits.
+
+    Durable Task (and therefore Dapr Workflows) creates its gRPC channels via
+    ``durabletask.internal.shared.get_grpc_channel``.  This helper monkey patches
+    that factory so that subsequent runtime/client instances honour the provided
+    ``grpc.max_send_message_length`` / ``grpc.max_receive_message_length`` values.
+    """
+    if not options:
+        return
+    if not options.max_send_message_length and not options.max_receive_message_length:
```

Comment:
can you update so folks can update one and not the other field and the options still apply pls?

Reply:
Good catch! The original logic already supported setting one or both options, but the double-negative check (if not X and not Y) was confusing. I've updated it to explicitly check is None for clarity and added a note in the docstring that users can set either or both options. Now it's clear that you can configure just `max_send_message_length`, just `max_receive_message_length`, or both independently.